packages/flutter/lib/src/material/slider.dart

```diff
@@ -124,6 +125,8 @@ class Slider extends StatefulWidget {
     this.activeColor,
     this.inactiveColor,
     this.semanticFormatterCallback,
+    this.focusNode,
```

Comment:
Is RangeSlider going to also get this treatment after?

Reply:
Yes, but I think it will be a little more tricky: https://github.com/flutter/flutter/issues/53958